guides/04_additional-features/03_streaming-inputs.md

```diff
@@ -0,0 +1,59 @@
+# Streaming inputs
+
+In the previous guide, we covered how to stream a sequence of outputs from an event handler. Gradio also allows you to stream images from a user's camera or audio chunks from their microphone **into** your event handler. This can be used to create real-time object detection apps or conversational chat applications with Gradio.
+
+Currently, the `gr.Image` and the `gr.Audio` components support input streaming via the `stream` event.
+Let's create the simplest streaming app possible, which simply returns the webcam stream unmodified.
+
+$demo_streaming_simple
+
+Try it out! The streaming event is triggered when the user starts recording. Under the hood, the webcam will take a photo every 0.1 seconds and send it to the server. The server will then return that image.
+
+There are two unique keyword arguments for the `stream` event:
+
+* `time_limit` - This is the amount of time the gradio server will spend processing the event. Media streams are naturally unbounded so it's important to set a time limit so that one user does not hog the Gradio queue. The time limit only counts the time spent processing the stream, not the time spent waiting in the queue. The orange bar displayed at the bottom of the input image represents the remaining time. When the time limit expires, the user will automatically rejoin the queue.
```

Comment:
Perhap include the the types, e.g. 
```suggestion
* `time_limit (float)` - This is the amount of time the gradio server will spend processing the event. Media streams are naturally unbounded so it's important to set a time limit so that one user does not hog the Gradio queue. The time limit only counts the time spent processing the stream, not the time spent waiting in the queue. The orange bar displayed at the bottom of the input image represents the remaining time. When the time limit expires, the user will automatically rejoin the queue.
```